src/workflow/custom-vision-model-v2/.package/Manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+    "modes": {
+        "batch-account" : {
```

Comment:
For this PR, we do not want to have dependency on AKS PR. Those referenced pipeline files are linked service files are not available in latest main.